Examples/CaseStudies/SwiftUICaseStudies/04-HigherOrderReducers-ReusableFavoriting.swift

```diff
@@ -78,7 +78,7 @@ extension Reducer {
             .cancellable(id: FavoriteCancelId(id: state.id), cancelInFlight: true)
 
         case let .response(.failure(error)):
-          state.alert = .init(title: .init(error.localizedDescription))
+          state.alert = .init(title: TextState(error.localizedDescription))
```

Comment:
Even with `@_disfavoredOverload` on the deprecated initializer, `LocalizedStringKey` is favored here over `TextState`, so `.init` produces a deprecation warning. That warning will go away when we fully remove the deprecated interface.